.github/workflows/tests.yml

```diff
@@ -0,0 +1,27 @@
+name: Run tests
+
+on: [pull_request, push]
+
+jobs:
+  test:
+
+    strategy:
+      fail-fast: false
+      matrix:
+        python: [3.6, 3.7, 3.8, 3.9, 3.10-dev, pypy-3.7]
+
+    name: ${{ matrix.python }}
+    runs-on: ubuntu-latest
```

Comment:
In the future this could now be trivially expanded to include testing on Windows and macOS too :eyes: